Microsoft.NET.Build.Containers/Label.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.NET.Build.Containers;
+
+internal sealed record Label(string name, string value);
```

Comment:
I understand why this is here, but it really feels unnecessary to me. Maybe my least-favorite C#-ism :D No action required, just wanted to log a not-so-silent protest

Reply:
To calm the protest down a bit, this will disappear in https://github.com/dotnet/sdk-container-builds/pull/341 after a bug fix :)